webui/react/src/pages/ModelDetails.tsx

```diff
@@ -292,14 +293,16 @@ const ModelDetails: React.FC = () => {
         body: { name: editedName },
         modelName,
       });
+      window.location.href = `/det${paths.modelDetails(editedName)}`;
```

Comment:
This line doesn't work (at least on Netlify). You should be able to do `routeToReactUrl(paths.modelDetails(editedName))` instead.